src/compute/src/render/join/delta_join.rs

```diff
@@ -571,10 +571,10 @@ where
     G::Timestamp: Lattice + crate::render::RenderTimestamp + Refines<T>,
 {
     match trace {
-        SpecializedArrangementImport::Bytes9Row(key_types, inner) => build_update_stream(
+        SpecializedArrangementImport::RowUnit(inner) => build_update_stream(
```

Comment:
Uncovered

Reply:
This should be hopefully covered now by a change in the test introduced in this PR. I focused on the cases unveiled by the coverage run, given that other paths in the PR were already covered by the existing tests.